src/Horse.Core.Route.Intf.pas

```diff
@@ -0,0 +1,44 @@
+unit Horse.Core.Route.Intf;
+
+interface
+
+uses
+  Horse.Router;
+
+type
+  IHorseCoreRoute = interface
+    ['{39ABE601-E812-40B4-8857-F14C938B9773}']
+    function Get(ACallback: THorseCallback):IHorseCoreRoute; overload;
```

Comment:
Change to in all functions:

`function Get(ACallback: THorseCallback): IHorseCoreRoute; overload;`